examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  dog plane plane plane\n"
+     ]
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "# functions to show an image\n",
+    "\n",
+    "def imshow(img):\n",
+    "    img = img / 2 + 0.5     # unnormalize\n",
+    "    npimg = img.numpy()\n",
+    "    plt.imshow(np.transpose(npimg, (1, 2, 0)))\n",
```

Comment:
Could you please change to use **plt.figure()** and **plt.subplot()** to draw a bigger picture with fixed size?
And just use **plt.title("XXX")** to set label name, your labels are not aligned with images.

Reply:
This is part of the original cifar-10 tutorial. Kept it like this so users can easily recognize.

Reply:
OK, let's keep it.